### DISCUSSION TITLE
zh/guide/drivers/sftp

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#183] [Announcements]
# zh/guide/drivers/sftp

即通过 SSH 协议访问文件。 地址 SSH 地址，需要包含端口。 用户名 SSH 用户名 密码 SSH 密码 私钥 使用私钥验证而不是密码。 根文件夹路径 挂载的根路径 默认使用的下载方式

https://alist.nn.ci/zh/guide/drivers/sftp.html

**Comment by tanxingeng** [comment]
sftp连接成功，但是断线后，不会自动重连，会一直处于断线状态，必须手工禁用后再启用，才能再次连上。

**Comment by jsfnzhangwei** [comment]
根文件夹设置无效，不管设置那个文件夹，进入都是根文件夹

**Reply by tuonisitake** [reply]
你后来解决了吗

**Reply by jsfnzhangwei** [reply]
我更新到v3.28.0后就可以了

**Comment by daqyt** [comment]
为什么我ssh连得上，但是在alist上不行

**Comment by ooyyee** [comment]
连接OpenWrt成功,但就是不能打开etc目录,打开他目录正常.

**Comment by Lipuyari** [comment]
想要连接的主机经过了跳板机怎么连接啊

**Comment by chenxiahuaxu** [comment]
alist管理局域网个内所有的sftp。
局域网是千兆速率，挂载alist的主机也是千兆，测试全是千兆达标的。
下载文件的速度却不足百兆。

**Comment by cloudskyic** [comment]
用cx文件管理器webdav alist挂载sftp，无法实时更新文件，需要alistweb界面刷新后才行
